src/object/srv_ec_aggregate.c

```diff
@@ -1579,12 +1579,11 @@ agg_peer_update(struct ec_agg_entry *entry, bool write_parity)
 			for (i = 0; i < failed_tgts_cnt; i++) {
 				if (targets[i].ta_comp.co_rank ==
 				    peer_loc->sd_rank) {
-					D_ERROR(DF_UOID" peer parity tgt "
-						"failed rank %d, tgt_idx %d.\n",
-						DP_UOID(entry->ae_oid),
+					D_DEBUG(DB_EPC, DF_UOID" peer parity "
```

Comment:
minor, ERR log level seems better.

Reply:
hmm, IMHO this is actually not error (expected),  if it knows the peer has been down. So error message will confuse the administrator or developer. So I changed it to DEBUG here.